src/player.cpp

```diff
@@ -5828,6 +5828,28 @@ static int bound_mod_to_vals(int val, int mod, int max, int min)
     return mod;
 }
 
+void player::print_health()
+{
+    if( !is_player() ) {
+        return;
+    }
+
+    if( healthy > 50 ) {
+        add_msg( m_good, _("You're very healthy, your wounds heal faster") );
```

Comment:
The messages are fine for the selfaware thing, but they could use some
personality for the wake-up message.
"You feel fantastic!  All that healthy living is doing you good."
"Your wounds are healing pretty fast, you must be taking good care of
yourself."
"You feel a little cruddy, getting out of bed is a struggle."
"You don't feel so good, not enough to slow you down, but at this rate
you're headed for a cold."
"You feel terrible, maybe you should think about eating better."
"Your wounds aren't healing as fast as they should, maybe some vitamins are
in order."
Ideally they'd be randomized a bit and situational, displaying the messages
about healing only when you aren't at full HP, but that's most of the time
I'd think.